components/02-elements/mega-menu/mega-menu.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="mega-menu">
+    <ul class="mega-menu__wrapper container">
+        {{#each list}}
+            <li class="mega-menu__item {{ class }}">
+                <a href="#" class="mega-menu__link">
+                    {{text}}
+                </a>
+
+                {{#if nestedList}}
+                    <ul class="
+                            mega-menu__dropdown
+                            mega-menu__dropdown--level1
+                        "
+                    >
+                        {{#each nestedList}}
+                            <li class="
+                                    mega-menu__dropdown-item
+                                    mega-menu__dropdown-item--level1
+                                    {{#if modifier}}
+                                        {{modifier}}
+                                    {{/if}}
+                                "
+                            >
+                                <a href="#"
+                                   class="
+                                       mega-menu__link
+                                       mega-menu__link--inner-heading
+                                   "
+                                >
+                                    {{ heading }}
+                                </a>
+
+                                {{> '@list--link'
+                                    listTag="ul"
+                                    elementTag="li"
+                                }}
```

Comment:
I am wondering if it shouldn't be a module if we use another element inside it? What do you think?

Reply:
moved to modules